xen/drivers/passthrough/arm/scmi_smc_maker.c

```diff
@@ -0,0 +1,350 @@
+/*
+ * scmi_process.c
+ *
+ *  Created on: Sep 3, 2024
+ *      Author: amoiseev
+ */
+
+#include <asm/p2m.h>
+#include <xen/device_tree.h>
+#include <xen/init.h>
+#include <xen/iocap.h>
+#include <xen/lib.h>
+#include <xen/libfdt/libfdt.h>
+#include <xen/scmi_smc_maker.h>
+
+#define SCMI_NODE_PATH_MAX_LEN 128
+#define SCMI_NODE_COMPATIBLE   "arm,scmi-smc"
+#define SCMI_SHMEM_COMPATIBLE  "arm,scmi-shmem"
```

Comment:
Could you drop these ^? They are informative by itself

Reply:
+